.github/workflows/php.yml

```diff
@@ -10,12 +10,20 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
```

Comment:
Do we even need to test it on PHP 7.2 if it reached the end of life almost year ago?
I would even recommend removing PHP 7.2 support from composer.json
cc: @vkublytskyi

Reply:
I'm ok with it, but currently, it is there, so it's better to make sure it is working with all declared versions

Reply:
I just added commit to remove PHP 7.2 support